README.md

```diff
@@ -69,6 +92,27 @@ Run the containers:
 
 Visit http://localhost:8000/petition/api/ in your browser.
 
+### Using docker-compose.override.yml
+
+To develop in a Docker container, we'll use a `docker-compose.override.yml`
+override file to configure the Django container to sleep by default:
+
+```yaml
+# file: docker-compose.override.yml
+version: '3'
+
+services:
+  django:
+    command: ["sleep", "infinity"]
+```
+
+Now we run `runserver` manually to have more control over restarts:
+
+```sh
+docker compose exec django bash
+root$ python manage.py runserver 0.0.0.0:8000
+```
```

Comment:
I don't think i'm fully understanding this section. Is this a new required step in getting the project to run? Is it just a suggestion?

Reply:
It's a suggestion. By default, `runserver` starts automatically in the `dev` stage:

https://github.com/deardurham/dear-petition/blob/8eb977bc56fdaa8bb4b88f914d161e90a8a4282c/Dockerfile#L150

However, this doesn't always fail cleanly when say a syntax error is introduced in a Python module and the file watcher attempts to auto-reload runserver. It may fail entirely and you have to re-run `docker compose up -d django up`. If you use the suggestion documented here though, the user manually runs runserver within the container, so if it fails, you can just restart the process.